### ISSUE TITLE
[FEAT] Publish Docker image to Docker Hub

### ISSUE THREAD

**Issue by jahstreet** [Netflix/nf-data-explorer#9]
**Which Datastore is this regarding?**
All

**Is your feature request related to a problem? Please describe.**
I couldn't find the Docker image available on Docker Hub. The challenge I face is to build the Docker image in a restricted env, requiring me to make all the dependencies needed for the build available. That would unlock the use of the project for me if I could simply pull it from the Docker Hub. Do you have that feature in plans?

**Describe the solution you'd like**
Make a Docker image available in Docker Hub.

**Describe alternatives you've considered**
Fork and do it myself.

**Additional context**
N/A

Bottom line, it complicates it for me a lot to pilot the tool, so I hesitate trying it out w/o it.
